src/filebrowser/browser.ts

```diff
@@ -106,10 +106,11 @@ class FileBrowserWidget extends Widget {
 
     model.refreshed.connect(this._handleRefresh, this);
     this._crumbs = new BreadCrumbs({ model });
+    this._listing = new DirListing({ manager, model, opener, renderer });
+    let listings = this._listing;
     this._buttons = new FileButtons({
-      commands, keymap, manager, model, opener
+      commands, keymap, manager, model, opener, listings
     });
-    this._listing = new DirListing({ manager, model, opener, renderer });
 
     model.fileChanged.connect((fbModel, args) => {
```

Comment:
I am trying to take care of this right now, but it doesn't seem like `args` even has a `.type` property. Is this something that is coming in a PR soon?